include/uv-win.h

```diff
@@ -647,3 +647,17 @@ RB_HEAD(uv_timer_tree_s, uv_timer_s);
 #ifndef X_OK
 #define X_OK 1
 #endif
+
+#if _MSC_VER >= 1900
+#undef stdin
```

Comment:
I still don't understand what you mean. Maybe paste some output. But I don't think there is a problem with libuv. We already build 2 applications linked with libuv: run-tests.exe and run-benchmarks.exe, and they build just fine. You are probably missing linking with some libraries, as @bnoordhuis pointed out.